evmrpc/tx_test.go

```diff
@@ -60,6 +60,7 @@ func TestGetTxReceipt(t *testing.T) {
 	require.Equal(t, "0x0123456789012345678902345678901234567890123456789012345678901234", resObj["transactionHash"].(string))
 	require.Equal(t, "0x0", resObj["transactionIndex"].(string))
 	require.Equal(t, "0x1", resObj["type"].(string))
+	require.Equal(t, "0x1234567890123456789012345678901234567890", resObj["contractAddress"].(string))
```

Comment:
nit - can we also add a test case for contractAddress = nil (to prevent potential future regressions)?

Reply:
yea added